script/deploy/deployAndStoreCREATE3Factory.sh

```diff
@@ -84,6 +85,9 @@ deployAndStoreCREATE3Factory() {
   mv "$tmpfile" "$NETWORKS_JSON_FILE_PATH"
   echo "Stored CREATE3Factory address ($FACTORY_ADDRESS) in networks.json for network \"$NETWORK\""
 
+  # verify CREATE3Factory
+  # verifyContract "$NETWORK" "CREATE3Factory" "$FACTORY_ADDRESS" "0x"
```

Comment:
should it be commented?

Reply:
yeah that's not working at the moment. I have a separate ticket for that.